README.md

```diff
@@ -672,6 +672,66 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+PUT local files to stage
+------------------------
+
+PUT command can be used to upload files of a local directory or a single local file to the Snowflake stages (named, internal table stage or internal user stage).
+Such staging files can be used to load data into a table. 
+More on this topic: [File staging with PUT](https://docs.snowflake.com/en/sql-reference/sql/put).
+
+In the driver the command can be executed in a bellow way:
+```cs
+using (IDbConnection conn = new SnowflakeDbConnection())
+{
+    try
+    {
+	    conn.ConnectionString = "<connecting parameter>";
+	    conn.Open();
+	    var cmd = (SnowflakeDbCommand)conn.CreateCommand(); // cast allows get QueryId from the command
+	
+	    cmd.CommandText = $"PUT file://some_data.csv @my_schema.my_stage AUTO_COMPRESS=TRUE";
+	    var reader = cmd.ExecuteReader();
+	    Assert.IsTrue(reader.read()); // on success
+        Assert.DoesNotThrow(() => Guid.Parse(cmd.GetQueryId()));
+    }
+    catch (SnowflakeDbException e)
+    {
+        Assert.DoesNotThrow(() => Guid.Parse(e.QueryId)); // when failed
+        Assert.That(e.InnerException.GetType(), Is.EqualTo(typeof(FileNotFoundException)));
```

Comment:
I'm thinking if all this asserts should be in documentation... Someone could be confused and get wrong conclusions like that always FileNotFound is the inner exception. Maybe we need to add a comment clarifying that it is just an example? Is it possible to have SnowflakeDbException without QueryId, for example if some prevalidation fails?